modules/avformat/avformat.c

```diff
@@ -368,7 +399,19 @@ void avformat_shared_set_video(struct shared *sh, struct vidsrc_st *st)
 
 static int module_init(void)
 {
-	int err;
+    int err;
+    char hwaccel[64] = "";
+
+    conf_get_str(conf_cur(), "avformat_hwaccel", hwaccel, sizeof(hwaccel));
+    if (strnlen(hwaccel, sizeof(hwaccel)) > 0) {
```

Comment:
you can also use `str_isset` to check if the string is set, or
check return value of conf_get_str.